src/platform/nrfconnect/BUILD.gn

```diff
@@ -75,6 +75,20 @@ static_library("nrfconnect") {
     }
   }
 
+  if (chip_enable_wifi) {
+    sources += [
+      "../Zephyr/WiFiManager.h",
```

Comment:
I think if we're planning to use WPA supplicant low-level API and not the Zephyr management API, we may want to put it in `nrfconnect` dir. Not sure it has already been decided though.

Reply:
This is not so clear yet :)